src/commands/serve.ts

```diff
@@ -61,7 +78,23 @@ export class ServeCommand implements Command {
       return env.serve(serverOptions);
     }
 
-    logger.debug('serving via polyserve.startServer()...');
-    return polyserve.startServer(serverOptions);
+    logger.debug('serving via polyserve.startServers()...');
+    const serverInfos = await startServers(serverOptions);
+
+    if (serverInfos.kind === 'mainline') {
+      const mainlineServer = serverInfos;
+      const urls = getServerUrls(options, mainlineServer.server);
+      console.log(`Files in this directory are available under the following URLs
+      applications: ${
+                    url.format(urls.serverUrl)}
+      reusable components: ${url.format(urls.componentUrl)}
+    `);
+    } else {
+      // We started multiple servers, just tell the user about the control server,
+      // it serves out human-readable info on how to access the others.
+      const urls = getServerUrls(options, serverInfos.control.server);
+      console.log(`Started multiple servers with different variants:
+      More info here: ${url.format(urls.serverUrl)}`);
```

Comment:
I don't love this wording, "More info here" feels like something went wrong / there's something I need to read to fix.

How about something shorter like: `'Started Polyserve on ${url.format(urls.serverUrl)}'`?